hackathon_site/dashboard/frontend/src/testing/mockData.tsx

```diff
@@ -606,8 +606,8 @@ export const mockPendingOrders: Order[] = [
         team_id: 2,
         team_code: "IEEE",
         status: "Ready for Pickup",
-        created_at: "2021-10-17T18:28:44.691969-04:00",
-        updated_at: "2021-12-03T23:01:46.606892-05:00",
+        created_at: "2020-10-17T18:28:44.691969-06:00",
```

Comment:
Do you think it would make more sense to convert the string to more readable format like "Updated at: February 19, 2023, 09:18 AM"?

Reply:
That would make it more readable. The time currently being retrieved is in Military time (ie: using the 24h clock). Not sure how to go about making this work, maybe I will have to create a conditional statement somewhere to handle all times starting from 13:00 onward, and process them accordingly

Reply:
You can try something like this 
```typescript
function formatDateTime(dateTimeString: string): string {
  const dateTime = new Date(dateTimeString);
  const options: Intl.DateTimeFormatOptions = {
    month: "long",
    day: "numeric",
    year: "numeric",
    hour: "numeric",
    minute: "numeric",
    hour12: true
  };
  return dateTime.toLocaleString("en-US", options);
}

const inputDateTime: string = "2020-10-17T18:28:44.691969-06:00";
const formattedDateTime: string = formatDateTime(inputDateTime);

console.log(formattedDateTime);

```